pkg/server/http/http.go

```diff
@@ -131,3 +138,7 @@ func registerProbes(mux *http.ServeMux, p *prober.HTTPProbe, logger log.Logger)
 		mux.Handle("/-/ready", p.ReadyHandler(logger))
 	}
 }
+
+func OfBool(i bool) *bool {
```

Comment:
Nit: Does this need to be exported? Could we add a small comment why it is needed?